RPIPs/RPIP-43.md

```diff
@@ -72,14 +72,15 @@ Node operators can manage the set of validators in their megapool:
 - The oDAO SHALL be able to increase `debt` to apply penalties (see [RPIP-42](./RPIP-42.md/#penalizable-offenses)) by majority vote
   - If this causes the megapool to meet the criteria for a non-node operator initiated exit (see [RPIP-44](RPIP-44.md)), it is RECOMMENDED that an exit is initiated by the same transaction 
 - There SHALL be a function to pay off `debt` with ETH. The ETH SHALL be sent to the rETH contract and the `debt` SHALL be reduced by the amount.
+^ does the smoothing pool need to be accounted for here if they are opted into it? ^
```

Comment:
I don't think so. debt isn't to other NOs in the smoothie, it's explicitly to rETH